src/driver/mysql/MysqlDriver.ts

```diff
@@ -345,7 +324,6 @@ export class MysqlDriver implements Driver {
     constructor(connection: DataSource) {
         this.connection = connection
         this.options = {
-            legacySpatialSupport: true,
```

Comment:
this is a breaking, wdyt about waiting for v1?

Reply:
I can also revert it (and only use it in the tests).

Will try to do this when I get some time.

Reply:
Sounds good to me